Library/Homebrew/cmd/audit.rb

```diff
@@ -676,6 +676,10 @@ def audit_text
     if text =~ /def plist/ && text !~ /plist_options/
       problem "Please set plist_options when using a formula-defined plist."
     end
+
+    if text =~ %r{require "language/go"} && text !~ /go_resource/
```

Comment:
It's 3am, so I could be in idiot mode. I probably am, to be fair. But `text.include?` is vomiting some errors for me in audit:
```ruby
if text.include?("def plist") && !text.include?("plist_options")
  problem "Please set plist_options when using a formula-defined plist."
end
```
```
~> brew audit a2ps
Error: undefined method `include?' for #<FormulaText:0x007ff6a21512d8>
Please report this bug:
    https://git.io/brew-troubleshooting
/usr/local/Library/Homebrew/cmd/audit.rb:682:in `audit_text'
/usr/local/Library/Homebrew/cmd/audit.rb:1006:in `audit'
/usr/local/Library/Homebrew/cmd/audit.rb:69:in `block in audit'
/usr/local/Library/Homebrew/cmd/audit.rb:65:in `each'
/usr/local/Library/Homebrew/cmd/audit.rb:65:in `audit'
/usr/local/Library/brew.rb:87:in `<main>'
```

It works outside of `audit` though if I drop into `brew irb`:
```ruby
text = "blob"
text.include?("blob") && !text.include?("blos")
=> true
```

In which way am I missing something obvious here?